src/estree.ts

```diff
@@ -415,12 +413,9 @@ export interface EmptyStatement extends _Node {
 export interface ExportAllDeclaration extends _Node {
   type: 'ExportAllDeclaration';
   source: Literal;
+  exported: Identifier | null;
 }
 
-export interface ExportNamespaceSpecifier extends _Node {
-  type: 'ExportNamespaceSpecifier';
```

Comment:
Has this AST node been removed from the ESTree specs?

Reply:
Yes. It is not in estree spec master anymore.

Reply:
I see. What about [AnonymousDefaultExportedFunctionDeclaration](https://github.com/estree/estree/blob/f9d7bd7e6b8d9476f2eaa71d6ca279dff8d49d82/spec.estree#L799) ?

Reply:
We seem handled those with normal FunctionDeclaration and ClassDeclaration.